docs/sources/static/set-up/install-agent-binary.md

```diff
@@ -1,32 +1,49 @@
 ---
-title: Install static mode on other systems
+title: Other systems
 weight: 140
 aliases:
 - ../../set-up/install-agent-binary/
 ---
 
-## Install static mode on other systems
+# Install Grafana Agent on other systems
```

Comment:
Will it be confusing if we don't mention what mode this is here? (static mode vs static mode operator or Flow mode)

Reply:
It works either way. The topic is in the static mode section... but it's easy to add static mode to the heading and this removes any question

Reply:
Yeah, that's true, I'm considering that the other install pages currently explicitly state what mode they're for, and whether that has an impact on SEO when trying to find the correct docs from Google.

I'll leave the final call up to you though, you definitely know more about this than I do :)

Reply:
fixed